### ISSUE TITLE
css custom properties

### ISSUE THREAD

**Issue by bkardell** [LeaVerou/css-almanac#22]
Last year we adjusted the data gathering so we could look at and examine custom elements, I even made a little tool that let you explore the dataset collections with regexps instantly because the dataset is rather specific and comparatively small (it's a loadable json file even of reasonable size)... This was really, really handy in exploring (we even found and helped fix a very popular bug because of it) and I've used it a number of times since - @rviscomi gives me a new dump of data occasionall and we can explore it over time even.... I wonder if we could do the same for css custom properties? I expect the same basic idea/data model would work?

**Comment by rviscomi** [comment]
Should be doable. See the relevant section in last year's chapter: https://almanac.httparchive.org/en/2019/css#custom-properties

> we even found and helped fix a very popular bug because of it

Do you remember what that was? I'm putting together a list of ways the 2019 Web Almanac had a positive impact on the community and this sounds like a perfect example.

**Comment by bkardell** [comment]
Yeah, when we were looking at the data and finding bugs by saying "hmm... this can't be right". One of our reported post-parse `.tagNames` to be like `div class=\"something very-specific unlikely-to-appear elsewhere\"` appearing on a thousands sites.  I asked you for some sample URLs to I could try to figure out what was happening and found they were all from the same industry service which provides turnkey websites for managing a popular kind of website (I don't want to shame anyone and name them) - it was some bad escaping.. I reached out to them and they found it and pushed a fix later that day.

**Comment by LeaVerou** [comment]
Related to #1

**Comment by bkardell** [comment]
Ah I would have sworn I didn't see it in the issues! Sorry @LeaVerou! 

Note: the data model will not work for many #1 questions - we had similar desires to know things about attributes but hard to generally collect and make easily queryable without running into just running a regexp over pages.. Maybe this is solvable tho and we were just not creative enough.  If so, I'd love to revist the q of attributes for markup.
